docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ApplyOnnxModel.cs

```diff
@@ -18,29 +18,17 @@ public static void Example()
             // Microsoft.ML.Onnx.TestModels nuget.
             var modelPath = @"squeezenet\00000001\model.onnx";
```

Comment:
Please verify that this statement is true:

1) Is the model.onnx in the github link the same as

Reply:
yep its there

---
In reply to: [275814495](https://github.com/dotnet/machinelearning/pull/3349#discussion_r275814495) [](ancestors = 275814495)